javascript/javascript2/week1/hyfBay.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function () {
```

Comment:
when have free time try to modularize function in  object method way
https://codepen.io/naresh699/pen/yoZZJe?editors=1010